spec/models/dependent_spec.rb

```diff
@@ -1,3 +1,27 @@
+# == Schema Information
+#
+# Table name: dependents
+#
+#  id                      :bigint           not null, primary key
+#  birth_date              :date
+#  disabled                :integer          default("unfilled"), not null
+#  first_name              :string
+#  last_name               :string
+#  months_in_home          :integer
+#  north_american_resident :integer          default("unfilled"), not null
+#  on_visa                 :integer          default("unfilled"), not null
+#  relationship            :string
+#  was_married             :integer          default("unfilled"), not null
+#  was_student             :integer          default("unfilled"), not null
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#  intake_id               :bigint           not null
+#
+# Indexes
+#
+#  index_dependents_on_intake_id  (intake_id)
+#
+
```

Comment:
Ah yeah, I've been getting this file generated locally as well. I think it might have gotten left behind on a previous commit. ✅